slides/package.json

```diff
@@ -8,6 +8,7 @@
     "clean": "rm -rf dist",
     "build": "npm run clean && webpack",
     "start": "webpack-dev-server",
+    "test": "cd testing && tsc index.ts --outdir dist -t es6 -m commonjs && cd dist && node index.js",
```

Comment:
I tried relative paths by running tsc directly like `tsc testing/index.ts`, but it caused problems with the relative paths to the slides that are defined in the code. We can sit down tomorrow together to see if this is we can fix.